RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
+    correctionPhaseNS = cms.double(0.0),
+    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
```

Comment:
This cfi and the corresponding algorithm in HcalSimpleReconstructor are just placeholders to demonstrate the use of the QIE10DigiCollection. This isn't in the RECO workflow for any scenario and never will be. Eventually it will be supplanted by the real algorithms, once they are developed.